app/views/content/is-teaching-right-for-me/who-do-you-want-to-teach.md

```diff
@@ -62,6 +68,7 @@ Once you have QTS, you're qualified to teach in primary and secondary schools. H
 The best way to work out which age group you would like to teach is to get real life experience in a classroom. 
 
 $get-school-experience$
+$experience$
```

Comment:
I'm not sure this is quite right - it's after the CTA to the GSE service so might be missed

I also wonder whether we should be pointing people to the GSE page on GIT here rather than straight to the GSE service - let's discuss!